README.md

```diff
@@ -1,2 +1,73 @@
-# declarative-custom-elements
-Explainer draft for specifying an approach to Declarative Custom Elements
+# Declarative Custom Elements
+### A discussion...
+
+## Problem
+Imperative JS code is required to unlock the power of [Custom Elements](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements), [Shadow DOM](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM), and other "web components" APIs. This is particularly true when these APIs are used in concert with each other. A declarative API would open the door for things like:
+
+- non-JS developers being given the power of custom elements
+- having custom element definitions available as parse time without requireing the jump over to the JS thread early in the document parse
+- others?
```

Comment:
leaving the multithreading/hydration/rendering optimization to the browser as no JS engaged.

Reply:
- enabling CMS to expose its content and templates as DCE

Reply:
* enabling static site builders without JS yet with rich set of components. Requires external template, which is not in proposal level yet.